modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,21 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+
+=== Spell Checker Pro 3.3.0
+
+The {productname} 6.5 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
+
+Spell Checker Pro 3.3.0 adds one change:
+
+=== Words containing non-Latin characters were not sent in spelling requests.
+//#TINY-9654
+
+As of Spell Checker Pro 3.2.1, only languages using a Latin-based alphabet, including English, French, German, etc., had supported dictionaries. As a result, Spell Checker Pro's client side only sent words it “understands” in spelling requests to the server, omitting words containing non-Latin characters from languages like Greek, Arabic, and Korean. Specifically, the word identification logic used the regular expression `/[\w'\\-\u00AD\u0100-\u017F\u00C0-\u00FF\uFEFF\u2018\u2019]/` to capture characters including the extended Latin alphabet. This interfered with the usages of some customers pairing the Spell Checker Pro client side with a custom spellchecking service with support for languages with non-Latin alphabets, as words containing non-Latin characters would not be sent to the server.
+
+In {productname} 6.5, the word identification logic for Spell Checker Pro's client side has now been changed to the one used by the Word Count plugin. This allows words containing non-Latin characters to be captured and sent in requests to the spelling service.
```

Comment:
```suggestion
=== Spell Checker Pro 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Spell Checker Pro** premium plugin.

Spell Checker Pro 3.3.0 adds one change:

=== Words containing non-Latin characters were not sent in spelling requests.
//#TINY-9654

As of Spell Checker Pro 3.2.1, only languages using a Latin-based alphabet — for example; English, French, and German — had supported dictionaries.

As a consequence, Spell Checker Pro’s client-side only sent strings containing Latin characters in spelling requests to the server, omitting words containing non-Latin characters from languages such as Greek, Arabic, and Korean.

This interfered with some use-cases in which Spell Checker Pro client-side was paired with a custom spellchecking service which supports languages with non-Latin alphabets: words containing non-Latin characters were not sent to the server.

In SpellChecker Pro 3.3.0, the word identification logic for the plugin’s client-side has been changed to the one used by the Word Count plugin. This allows words containing non-Latin characters to be captured and sent in requests to the spelling service.

For information on the **SpellChecker Pro** plugin, see: xref:introduction-to-tiny-spellchecker.adoc[SpellChecker Pro].
```

1. Incorporate @MitchCTiny’s [suggested edit](https://github.com/tinymce/tinymce-docs/pull/2708/files#r1228789115).
2. Copy edits.
3. Added the _for information_ line.

Reply:
@kemister85 This look good, let's commit this. Thanks @brian-forte @MitchCTiny